Ginger/Ginger/Run/RunSetActions/ExportResultsToALMConfigPage.xaml.cs

```diff
@@ -138,5 +154,78 @@ private void AttachActivitiesGroupReportCbx_Checked(object sender, RoutedEventAr
         {
             
         }
+
+        private void xChangeTestSetBtn_Click(object sender, RoutedEventArgs e)
+        {
+            GingerCoreNET.ALMLib.ALMConfig AlmConfig = null;
+            if (xALMTypeCbx.ComboBoxSelectedValue.ToString().Equals(RunSetActionPublishToQC.AlmTypeDefault))
+            {
+                AlmConfig = ALMIntegration.Instance.GetDefaultAlmConfig();
+            }
+            else
+            {
+                AlmConfig = WorkSpace.Instance.Solution.ALMConfigs.Where(alm => alm.AlmType.ToString().Equals(xALMTypeCbx.ComboBoxSelectedValue.ToString())).FirstOrDefault();
+            }
+            if (AlmConfig is null)
+            {
```

Comment:
Do we must do connection and fields set only because user changed the ALM type?
why not to do it only if user clicking on Set Fields button?